sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/ColumnDefinition.scala

```diff
@@ -80,6 +80,7 @@ case class ColumnDefinition(
         case _ => default.originalSQL
       }
       metadataBuilder.putString(EXISTS_DEFAULT_COLUMN_METADATA_KEY, existsSQL)
+      metadataBuilder.putString(StructField.SQL_FUNCTION_DEFAULT_METADATA_KEY, default.originalSQL)
```

Comment:
Does this mean table schema with default column value will have this unnecessary metadata?

Reply:
Yes. Is this a concern? If so, we can add this metadata in `visitSingleRoutineParamList` instead

Reply:
done